app/src/layouts/app-layout/AppLayout.tsx

```diff
@@ -5,17 +5,20 @@ import { useTranslation } from "react-i18next";
 
 import { ToastContextController } from "context/toast/ToastContextController";
 import { EVMWalletSelectorContextController } from "context/evm-wallet-selector/EVMWalletContextController";
+import { useLocalStorage } from "hooks/useLocalStorage/useLocalStorage";
 
 import { AppLayoutProps } from "./AppLayout.types";
 
 export const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
   const { t } = useTranslation("head");
   const { locale } = useRouter();
+  const localStorage = useLocalStorage();
 
   useEffect(() => {
     // @todo set with a toggle button from navbar or footer
-    document.body.dataset.theme = "light";
-  }, []);
+    const localTheme = localStorage.get<string>("theme");
+    document.body.dataset.theme = localTheme || "dark";
+  }, [localStorage]);
```

Comment:
I think we can remove this theme handling in `AppLayout` since a similar logic happens in the `ThemeSelector` component.